src/components/Evolution.ts

```diff
@@ -35,16 +84,23 @@ export class Evolution<TGameStartr extends FullScreenPokemon> extends Component<
      * Holds evolution requirement checks, keyed by the method of evolution.
      */
     private readonly requirementHandlers: IRequirementHandlers = {
-        level: (pokemon: IPokemon, requirements: IPokemonEvolutionByLevel): boolean => {
-            return pokemon.level >= requirements.level;
+        level: (args: IHandlerArgs): boolean => {
+            return args.pokemon.level >= (args.requirement as IPokemonEvolutionByLevel).level;
         },
-        item: (): boolean => {
-            // Need some way to pass a flag for the item being used
-            // Item usage outside of battle does not seem to be implemented (Issue #364)
+        item: (args: IHandlerArgs): boolean => {
+            if (args.modifier) {
+                if ((args.requirement as IPokemonEvolutionByItem).item === (args.modifier as IItemModifier).item.join("")) {
+                    return true;
+                }
+            }
             return false;
         },
-        trade: (): boolean => {
-            // Need some way to pass a flag in that the pokemon has just been traded
+        trade: (args: IHandlerArgs): boolean => {
+            if (args.modifier) {
+                if (args.modifier.type === "trade") {
+                    return true;
+                }
+            }
```

Comment:
This part is more something of a placeholder for when held items are implemented, as an additional check will need to be made here for held items... Should I change it to a one-liner for now?